csrfguard/src/main/java/org/owasp/csrfguard/servlet/JavaScriptServlet.java

```diff
@@ -48,6 +48,8 @@
 import java.util.Set;
 import java.util.regex.Pattern;
 
+import sun.nio.cs.StandardCharsets;
```

Comment:
Don't use packages under "sun.*" because they are not part of the supported public interface.

https://www.oracle.com/java/technologies/faq-sun-packages.html

You could use java.nio.charset.StandardCharsets#UTF_8 instead, but only in Java 1.7+.

Reply:
Oops. Should have cleaned that up. Wanted to use the constant but it's not in Java8 and then didn't revert the import. Will cleanup.